variod.c

```diff
@@ -317,9 +318,12 @@ int main(int argc, char *argv[])
 				break;
 			}
 			//Send the message back to client
-			//printf("SendNMEA: %s",client_message);
+			//fprintf(fp_console,"SendNMEA: %s",client_message);
 			// Send NMEA string via socket to XCSoar
-			if (send(xcsoar_sock, client_message, strlen(client_message), 0) < 0)
+			int sendbytes;
+			sendbytes=send(xcsoar_sock, client_message, strlen(client_message), 0);
+			fprintf (fp_console,"send command returned: %d\n",sendbytes);
```

Comment:
I think you can remove this `fprintf` in final version to avoid spamming the log

Reply:
Looked for it but it forgot to remove it.